hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketCreateRequest.java

```diff
@@ -201,6 +201,8 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
       // Add default acls from volume.
       addDefaultAcls(omBucketInfo, omVolumeArgs);
 
+      // quotaAdd used namespace
```

Comment:
How about we update it this to `add quota of used namespace` or the same as `update used namespace for volume` in `OMBucketDeleteRequest`  ?

Reply:
used `update used namespace for volume` for both now.